examples/source_examples/accelo_api_connector_multithreading_enabled/configuration.json

```diff
@@ -0,0 +1,5 @@
+{
+  "deployment": "<DEPLOYMENT>",
+  "client_id": "<CLIENT_ID>>",
```

Comment:
Replace <CLIENT_ID> with <YOUR_CLIENT_ID>
We follow similar pattern for all the configuration.json

Reply:
Done.